extension/src/popup/components/manageAssets/ChooseAsset/index.tsx

```diff
@@ -93,7 +93,14 @@ export const ChooseAsset = ({ balances }: ChooseAssetProps) => {
             domain: "",
           });
 
-          if (isExperimentalModeEnabled && sorobanBalances.length) {
+          // disable soroban payment until we can get the new SDK working
+          const disableSorobanPayment = true;
```

Comment:
cheap feature flag. `prepareTransaction` in the `handleXfer` is breaking and I'm not sure why. I'll try to get to the bottom of it today, but if not, let's just disable the flow for now.